cmd/gendeepcopy/deep_copy.go

```diff
@@ -61,7 +61,7 @@ func pkgPath(group, version string) string {
 		group = "api"
 	}
 	gv := group
-	if version != "" {
+	if version != "__internal" {
```

Comment:
It can't take a dependency on runtime here unfortunately.

On Mon, Jan 4, 2016 at 3:57 PM, David Eads <notifications@github.com> wrote:

> In cmd/gendeepcopy/deep_copy.go
> <https://github.com/kubernetes/kubernetes/pull/17922#discussion_r48780174>
> :
>
> > @@ -61,7 +61,7 @@ func pkgPath(group, version string) string {
> >  		group = "api"
> >  	}
> >  	gv := group
> > -	if version != "" {
> > +	if version != "__internal" {
>
> "__internal" should be a constant.
>
> —
> Reply to this email directly or view it on GitHub
> <https://github.com/kubernetes/kubernetes/pull/17922/files#r48780174>.
>